di-ipv-evcs-stub/lambdas/src/domain/vcMetadata.ts

```diff
@@ -0,0 +1,5 @@
+export default interface VCMetadata {
```

Comment:
Can this be dropped now that we're treating the incoming metadata as just an object?

I see that it's still being used in the `evcsVcItem`

Reply:
done